src/components/mobile/AnalyzeMobile.tsx

```diff
@@ -66,7 +68,7 @@ export default function AnalyzeMobile() {
             size={"lg"} />
         </Sidebar>
         <Sidebar.Pusher className="fill flex">
-        <Charts />
+          {/* <Charts /> */}
```

Comment:
let's leave this in for now. will need to rework/investigate further but afaik the analyze tab on prod mobile is just as bad as staging mobile